### PR TITLE
Add separate page for converting docker images to apptainer images

### PR DESCRIPTION
**Pull request by xamberl**
also added a section for building on local apptainer images

### CODE REVIEW

**Comment by xamberl** [comment]
Preview link: https://chtc.github.io/web-preview/preview-docker-to-apptainer/uw-research-computing/htc-docker-to-apptainer

File: _uw-research-computing/apptainer-build.md
Comment (aowen-uwmad) [inline _uw-research-computing/apptainer-build.md:1]: Good suggestion!

File: _uw-research-computing/htc-docker-to-apptainer.md
Comment (aowen-uwmad) [inline _uw-research-computing/htc-docker-to-apptainer.md:29]:
```suggestion
You will need to launch an interactive job to build the Apptainer image. Below is an example of a submit file. You may want to change memory or disk space requirements, depending on the expected requirements for your image.
```

File: _uw-research-computing/htc-docker-to-apptainer.md
Comment (aowen-uwmad) [inline _uw-research-computing/htc-docker-to-apptainer.md:40]:
```suggestion
```

Comment (aowen-uwmad) [reply _uw-research-computing/htc-docker-to-apptainer.md:40]: Not only has the bug been patched, we no longer need to specify the executable file. 
(You may want to confirm this before merging)

**Review by aowen-uwmad — APPROVED** [review]
Some suggestions, good to merge when you are ready.

**Review by ChristinaLK — APPROVED** [review]
lgtm. 1-2 things to check.

File: _uw-research-computing/apptainer-build.md
Comment (xamberl) [reply _uw-research-computing/apptainer-build.md:105]: I left the instructions on this page

**Pull request merged**
